.github/workflows/test.yml

```diff
@@ -12,6 +12,7 @@ jobs:
   lints:
     name: Format + Lints
     runs-on: ubuntu-latest
+    timeout-minutes: 10
```

Comment:
Looking at the last successful run on master, one of the jobs did take 11mins:
https://github.com/google/webcrypto.dart/actions/runs/12068757318

So maybe we should set the timeouts to 15mins.

Though I agree, it would be nice if we could engineer the jobs to finish faster.
Then again, we don't want to have too many jobs, it can be a resource waste and even slow because we'd have to wait for the jobs to start.

Reply:
[bfe006b](https://github.com/google/webcrypto.dart/pull/179/commits/bfe006bd2238b97d2aa24bad3c550ca3374296dd) 

Updated at 15 minutes.
Splitting the job will make it shorter than 10 minutes, but it is certainly possible to go over due to the effects of increasing the number of tests, etc. I think 15 minutes is reasonable. Thanks.